packages/pg-gateway/test/util.ts

```diff
@@ -128,3 +130,45 @@ export class DisposablePgClient extends Client {
     await this.end();
   }
 }
+
+export async function createPostgresClient(config: string | ClientConfig) {
```

Comment:
as we're now using these helpers in multiple test files, I've put them in utils.